src/subdomains/supporting/payment/services/transaction-helper.ts

```diff
@@ -109,8 +107,11 @@ export class TransactionHelper implements OnModuleInit {
     const outSpec = this.specRepo.getSpec(this.transactionSpecifications, toSystem, toAsset, TransactionDirection.OUT);
 
     return {
-      minFee: { amount: outSpec.minFee + inSpec.minFee, asset: 'EUR' },
-      minDeposit: { amount: Math.max(outSpec.minVolume, inSpec.minVolume), asset: 'EUR' },
+      minFee: { amount: outSpec.minFee + inSpec.minFee, asset: 'CHF' },
+      minDeposit: {
+        amount: Math.max(outSpec.minVolume, inSpec.minVolume),
+        asset: 'CHF',
+      },
```

Comment:
Do we need to switch to CHF in buy/sell/crypto controllers too?
![image](https://github.com/DFXswiss/api/assets/85513542/46d44e2f-71e6-44f4-b8f1-8b2832d65259)